docs/sources/developers/kinds/composable/phlaredataquery/schema-reference.md

```diff
@@ -0,0 +1,22 @@
+---
+keywords:
+  - grafana
+  - schema
+title: PhlareDataQuery kind
+---
+> Both documentation generation and kinds schemas are in active development and subject to change without prior notice.
+
+# PhlareDataQuery kind
+
+## Maturity: experimental
+## Version: 0.0
+
+## Properties
+
+| Property        | Type     | Required | Description                                         |
+|-----------------|----------|----------|-----------------------------------------------------|
+| `groupBy`       | string[] | No       | Allows to group the results.                        |
+| `labelSelector` | string   | No       | Specifies the query label selectors. Default: `{}`. |
+| `profileTypeId` | string   | No       | Specifies the type of profile to query.             |
```

Comment:
Filed an issue to fix docs generator to properly render required attributes https://github.com/grafana/schematization-and-as-code-project/issues/57

Reply:
Thank you!

Reply:
Pointed issues with docs generation [will fixed sooner than later](https://github.com/grafana/grafana/pull/62558).